retriever/lib/install.py

```diff
@@ -17,9 +15,6 @@ def _install(args, use_cache, debug):
     engine.use_cache = use_cache
 
     script_list = SCRIPT_LIST()
-    if not (script_list or os.listdir(SCRIPT_WRITE_PATH)):
-        check_for_updates()
-        script_list = SCRIPT_LIST()
```

Comment:
Should not download all the scripts if no offline script is present.

Reply:
Same as above, we can just leave it as downloading everything for now.